Wireframe/style.css

```diff
@@ -1,89 +1,92 @@
-  --line: 1px solid;
-  --container: 1280px;
-}
-/* ====== Base Elements ======
- General rules for basic HTML elements in any context */
-body {
-  background: var(--paper);
-  color: var(--ink);
-  font: var(--font);
-}
-a {
-  padding: var(--space);
-  border: var(--line);
-  max-width: fit-content;
-}
-img,
-svg {
-  width: 100%;
-  object-fit: cover;
-}
-/* ====== Site Layout ====== 
-Setting the overall rules for page regions
-https://www.w3.org/WAI/tutorials/page-structure/regions/
-*/
-main {
-  max-width: var(--container);
-  margin: 0 auto calc(var(--space) * 4) auto;
-}
-footer {
-  position: fixed;
-  bottom: 0;
-  text-align: center;
-}
-/* ====== Articles Grid Layout ====
-Setting the rules for how articles are placed in the main element.
-Inspect this in Devtools and click the "grid" button in the Elements view
-Play with the options that come up.
-https://developer.chrome.com/docs/devtools/css/grid
-https://gridbyexample.com/learn/
-*/
-main {
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  gap: var(--space);
-  > *:first-child {
-    grid-column: span 2;
-  }
-}
-/* ====== Article Layout ====== 
-Setting the rules for how elements are placed in the article. 
-Now laying out just the INSIDE of the repeated card/article design.
-Keeping things orderly and separate is the key to good, simple CSS.
-*/
-article {
-  border: var(--line);
-  padding-bottom: var(--space);
-  text-align: left;
-  display: grid;
-  grid-template-columns: var(--space) 1fr var(--space);
-  > * {
-    grid-column: 2/3;
-  }
-  > img {
-    grid-column: span 3;
-  }
-}
+/* Here are some starter styles
+You can edit these or replace them entirely 
+It's showing you a common way to organise CSS 
+And includes solutions to common problems 
+As well as useful links to learn more */
+
+/* ====== Design Palette ======
+ This is our "design palette".
+ It sets out the colours, fonts, styles etc to be used in this design 
+ At work, a designer will give these to you based on the corporate brand, but while you are learning
+ You can design it yourself if you like
+ Inspect the starter design with Devtools
+ Click on the colour swatches to see what is happening
+ I've put some useful CSS you won't have learned yet
+ For you to explore and play with if you are interested
+ https://web.dev/articles/min-max-clamp
+ https://scrimba.com/learn-css-variables-c026
+====== Design Palette ====== */
+:root {
+  --paper: oklch(7 0 0);
+  --ink: color-mix(in oklab, rgb(201, 123, 39) (--color) 5%, black);
+  --font: 100%/1.5 system-ui;
+  --space: clamp(6px, 6px + 2vw, 15px);
+  --line: 1px solid;
+  --container: 1280px;
+}
+/* ====== Base Elements ======
+ General rules for basic HTML elements in any context */
+body {
+  background:rgb(168, 205, 168);
+  color: rgb(25, 202, 25) (--ink);
+  font-family: "Oswald", sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  font-size:large;
+}
+a {
+  padding: 0px 50px 75px 100px (--space);

```

Comment:
It looks to me like you don't understand how variables are used in CSS, but you have asked no questions in your PR template. 

Would you like to know how to use them?